packages/devtools_app/lib/src/inspector/inspector_screen.dart

```diff
@@ -296,3 +371,72 @@ class InspectorScreenBodyState extends State<InspectorScreenBody>
     });
   }
 }
+
+class ErrorNavigator extends StatelessWidget {
+  const ErrorNavigator(
+      {Key key,
+      @required this.selectedErrorIndex,
+      @required this.errorCount,
+      @required this.onSelectedErrorIndexChanged})
+      : super(key: key);
+
+  final int selectedErrorIndex;
+  final int errorCount;
+  final void Function(int) onSelectedErrorIndexChanged;
+
+  @override
+  Widget build(BuildContext context) {
+    final label = selectedErrorIndex != null
+        ? 'Error ${selectedErrorIndex + 1}/$errorCount'
+        : 'Errors: $errorCount';
+    return Container(
+      color: devtoolsError,
+      child: Padding(
+        padding: const EdgeInsets.symmetric(horizontal: 12, vertical: 6),
+        child: Row(
+          children: [
+            Padding(
+              padding: const EdgeInsets.only(right: 6),
+              child: Text(label),
+            ),
+            IconButton(
+              padding: EdgeInsets.zero,
+              constraints: const BoxConstraints(),
+              splashRadius: Material.defaultSplashRadius / 2,
+              icon: const Icon(Icons.chevron_left),
```

Comment:
can we use `Icons.keyboard_arrow_up` and `Icons.keyboard_arrow_down` to match the navigators for Search in other parts of DevTools?
![Screen Shot 2021-02-08 at 9 25 39 AM](https://user-images.githubusercontent.com/43759233/107257622-b635a000-69ef-11eb-8050-4ff0deb42336.png)